apps/server/src/shared/repo/task/task.repo.ts

```diff
@@ -88,7 +91,20 @@ export class TaskRepo extends BaseRepo<Task> {
 		scope.addQuery(allForFinishedCoursesAndLessons.query);
 		scope.addQuery(allForCreator.query);
 
-		const countedTaskList = await this.findTasksAndCount(scope.query, options);
+		let { query } = scope;
+
+		if (filters?.userId) {
+			const forAssignedUser = new TaskScope();
+			forAssignedUser.byAssignedUser(filters.userId);
+
+			const filtersScope = new TaskScope('$and');
+			filtersScope.addQuery(forAssignedUser.query);
+			filtersScope.addQuery(scope.query);
+
+			query = filtersScope.query;
```

Comment:
not sure how to handle what github is complaining about here

Reply:
My linter is not complaining. Idk